pkg/client.go

```diff
@@ -187,6 +188,18 @@ func (c *Client) HandlePRRE(cfg PRBlockerConfig, pre *gh.PullRequestReviewEvent)
 
 	c.prLabels = ParseGHLabels(pr.Labels)
 
+	if pre.Review != nil {
+		if strings.ToLower(pre.Review.GetState()) == "approve" {
+			users := []*gh.User{
+				pre.Review.GetUser(),
+			}
+			err := c.Unassign(context.TODO(), owner, repoName, prNumber, users)
+			if err != nil {
+				return err
```

Comment:
In the `Assign()` case we log an `Info` message here. Do we want to do the same in the `Unassign()` case?

Reply:
Good point, I added a log in that case as well.